lib/database_config.rb

```diff
@@ -1,3 +1,11 @@
+##
+#
+# This module is intended *only* to manage database.yml
+# It is not accurate if DATABASE_URL is used, therefore
+# do not use this module outside of database.yml
```

Comment:
Still confused by this. `DatabaseConfig` *is* accurate when `DATABASE_URL` is used, no? Isn't that the point of `database_url_or_fallback`? My previous review I talked about how I thought your intention was just to keep Rails as the one source of truth, so this comment is a little misleading if that's the case. Maybe just say 'It is preferred that you use Rails' mechanisms directly in application code and leave this only for database.yml' or similar.

Reply:
I think I confused myself by not updating my description in the last PR.
Looking back, I was a bad developer and didn't change the PR to reflect the changes we had talked about.

Then since I didn't touch this for a bit, I completely forgot I had made those changes